.gitlab-ci.yml

```diff
@@ -305,15 +305,21 @@ step-1_deploy-prod-minor-dcs:
   extends:
     - .deploy-prod
   variables:
-    UPLOAD_PATH: eu1,us3,us5,ap1
+    UPLOAD_PATH: us3,us5,ap1
 
-step-2_deploy-prod-us1:
+step-2_deploy-prod-eu1:
+  extends:
+    - .deploy-prod
+  variables:
+    UPLOAD_PATH: eu1
+
+step-3_deploy-prod-us1:
   extends:
     - .deploy-prod
   variables:
     UPLOAD_PATH: us1
 
-step-3_deploy-prod-root:
+step-4_deploy-prod-root:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏publish-npm should be step 5 then